lib/types.ts

```diff
@@ -16,6 +16,7 @@ export type Profile = {
   userId: string;
   pictureUrl: string;
   statusMessage: string;
+  language: string;
```

Comment:
It should be`language?: string` because it only returns the language property when one of these situations applies:

- User has a LINE account created in Japan and has agreed to the Privacy Policy of LINE version 8.0.0 or later
- User has a LINE account created in Taiwan, Thailand, or Indonesia and has agreed to the Privacy Policy of LINE version 8.9.0 or later

Reply:
Make sense. Just corrected. Thank you @chentsulin!